conf/messages

```diff
@@ -1136,7 +1136,7 @@ paymentAllocation.tableHead.allocated-date                      = Date
 paymentAllocation.tableHead.amount                              = Amount
 paymentAllocation.taxYear										= Tax year {0} to {1}
 paymentAllocation.creditOnAccount                               = Credit on account
-paymentAllocation.moneyOnAccount                                = Money in your account
+paymentAllocation.moneyOnAccount                                = Remaining amount (added to available credit)
 paymentAllocation.moneyOnAccount.agent                          = Money in your client’s account
```

Comment:
This message was not requested to be changed. However it seems like it should as it is the agent counterpart to the message that was requested.

Reply:
Yeah I agree, we do display this content "money in your client's account" so worth going back to the BA and asking if it should be replaced

Reply:
It has been agreed to make the agent message the same as `moneyOnAccount`. Will make the change now.